jenkins/run-pytests

```diff
@@ -23,5 +23,11 @@ pytest_sts=${?}
 if [[ ${pytest_sts} -ne 0 ]]; then
     printf -- "FAILED: pytest ...\n" >&2
 fi
-final_sts=$(( black_sts + flake_sts + pytest_sts))
+printf -- "\nPython coverage ...\n"
+coverage report
```

Comment:
If we don't have `pytest --cov`, or the like, above, from what coverage data will this report be generated?

Reply:
From the entire code base, we are running all the unit tests anyways.

Reply:
How do we run the unit tests with coverage if we are not using `pytest --cov`?  See also PR https://github.com/zulcss/pbench/pull/3.